src/main/java/com/powsybl/openloadflow/sensi/apiv2/MatrixSensitivityFactor.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.sensi.apiv2;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.powsybl.math.matrix.DenseMatrix;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class MatrixSensitivityFactor {
+
+    private final List<String> functionsIds;
+
+    private final FunctionType functionType;
+
+    private final List<String> variablesIds;
```

Comment:
At some point, do we want to be able to have a matrix of MultiVariables ? For example, if you want to compute multiple GLSKs on the same branches. In which case, I guess this List<String> would be replaced by a List<Variable>